### ISSUE TITLE
Bug: qmd and jupyter cache

### ISSUE THREAD

**Issue by ludwigHoon** [quarto-dev/quarto-cli#185]
I recently encountered a peculiar situation where cached execution in qmd causes error during render. 

Example: 

````
---
title: Test
toc: false
---
```{python}
print("HELLO")
```
````

Running quarto render produces the following error: 
The execution_count just keeps incrementing. 
![image](https://user-images.githubusercontent.com/30918181/141947561-0eba2802-1f1e-4ce1-ad79-baf8d84dfde2.png)

Meanwhile, specifying: 
```
execute:
    cache: false
```
produces no error. 

`cache-refresh` does not affect either run.

**Comment by jjallaire** [comment]
Thanks, I can repro this locally as well. I'm guessing something changed in an update to jupyter-cache (noticed that the way they serialized notebooks seems to have changed). Will run this down and hopefully push a fix shortly!

**Comment by jjallaire** [comment]
This should fix the problem: https://github.com/quarto-dev/quarto-cli/commit/e044e6b57df4b565e07ce15946edfed15a87b13a

Fix is available in v0.2.283 --- LMK if this resolves on your end

**Comment by ludwigHoon** [comment]
Awesome, worked like a charm, thanks!
